### PR TITLE
Update storiesTemplate

### PR DESCRIPTION
**Pull request by danielle-dsouza**
Updated the `stories.tsx` template to account for what I've learned about stories during my recent PR reviews!

🤔 Regarding what you said about the comments, I'm also not sure whether we still need them since they are really extensive and there's lots of online tutorials/documentation for how to write stories, but regardless I figured I wouldn't touch them in this PR.

### CODE REVIEW

File: src/templates/storiesTemplate.ts [hunk 1]
```diff
@@ -62,7 +48,19 @@ export function storiesTemplate(
...
+    `};\n\n` +
+
+    `Basic.argTypes = {\n` +
+    ` exampleSelect: {\n` +
+    `   control: {\n` +
+    `     type: 'select',\n` +
+    `     options: ['option1', 'option2'],\n` +
+    `   },\n` +
+    ` },\n` +
+    `};\n\n`;
```
[the remark this answers is not in the document]
Comment (andrewmcgov) [reply src/templates/storiesTemplate.ts:63]: I like this! It demonstrates how to quickly add controls to a story.

File: src/templates/storiesTemplate.ts [hunk 2]
```diff
@@ -40,20 +39,7 @@ export function storiesTemplate(
...
-  text =
-    text +
-    `const Template: StoryFn<${componentName}Props> = (args) => <${componentName} {...args} />;\n\n`;
-
-  if (verboseComments) {
-    text = text + `// 👇 Each story then reuses that template\n`;
-  }
-
-  text = text + `export const Basic = Template.bind({});\n\n`;
+    `export const Basic: StoryFn<Props> = (args) => <${componentName} {...args} />;\n\n`;
```
[the remark this answers is not in the document]
Comment (andrewmcgov) [reply src/templates/storiesTemplate.ts:42]: Im leaning towards leaving the template in there, so it's there if you need it. I don't think it adds too much overhead when its not needed, what do you think?

**Review by andrewmcgov — COMMENTED** [review]
Thanks for adding what you've learned here! Just left a couple comments

File: src/templates/storiesTemplate.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (danielle-dsouza) [reply src/templates/storiesTemplate.ts:42]: Fair point!

File: src/templates/storiesTemplate.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielle-dsouza) [reply src/templates/storiesTemplate.ts:63]: 🥳

File: src/templates/storiesTemplate.ts [hunk 3]
```diff
@@ -5,8 +5,7 @@ export function storiesTemplate(
   let text =
     `import React from 'react';\n` +
     `import {Meta, StoryFn} from '@storybook/react';\n\n` +
-    `import type {${componentName}Props} from './${componentName}';\n` +
-    `import {${componentName}} from './${componentName}';\n\n`;
+    `import {${componentName}, Props} from './${componentName}';\n\n`;
```
Comment (andrewmcgov) [inline src/templates/storiesTemplate.ts:8]: We added the separate `type` import back in https://github.com/andrewmcgov/react-component-generator/pull/15. 

Was there a reason this was changed?

**Review by andrewmcgov — COMMENTED** [review]
Nice! Just one comment / request. We can ship this next week!

**Comment by andrewmcgov** [comment]
@danielle-dsouza can you also add a point about your change in `CHANGELOG.md` under the `Unreleased` heading?

**Pull request closed without merging**